src/NLog/Config/LoggingConfigurationParser.cs

```diff
@@ -1227,19 +1223,7 @@ private static string StripOptionalNamespacePrefix(string attributeValue)
 
             return attributeValue.Substring(p + 1);
         }
-
-        /// <summary>
-        /// Remove all spaces, also in between text. 
-        /// </summary>
-        /// <param name="s">text</param>
-        /// <returns>text without spaces</returns>
-        /// <remarks>Tabs and other whitespace is not removed!</remarks>
-        private static string CleanSpaces(string s)
```

Comment:
funny, this is something else then a trim. Not sure why this was introduced.

Reply:
To remove trailing spaces in `Info, Warn `